platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/problem/InternalProblemAwareFailure.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.tooling.internal.protocol.problem;
+
+import org.gradle.tooling.internal.protocol.InternalBasicProblemDetailsVersion3;
+import org.gradle.tooling.internal.protocol.InternalFailure;
+
+import java.util.List;
+
+/**
+ * DO NOT CHANGE THIS INTERFACE. It is part of the cross-version protocol.
+ *
+ * @since 8.11
+ */
+public interface InternalProblemAwareFailure extends InternalFailure {
+
+    List<InternalBasicProblemDetailsVersion3> getProblems();
```

Comment:
🤔 So we can't use a subtype here, right?

Reply:
The interface no longer exists,